packages/react-components/lib/tasks/create-task.tsx

```diff
@@ -1851,7 +914,9 @@ export function CreateTaskForm({
                       setOpenFavoriteDialog(true);
                     }}
                     style={{ marginTop: theme.spacing(2), marginBottom: theme.spacing(2) }}
-                    disabled={taskDefinitionId === 'custom_compose'}
+                    disabled={
+                      taskDefinitionId === DefaultCustomComposeTaskDefinition.taskDefinitionId
```

Comment:
Document why custom compose is not allowed as a favorite task.

Reply:
There is already a FIXME in the code